letsencrypt/display/ops.py

```diff
@@ -74,7 +74,7 @@ def pick_plugin(config, default, plugins, question, ifaces):
 
     if len(prepared) > 1:
         logger.debug("Multiple candidate plugins: %s", prepared)
-        plugin_ep = choose_plugin(prepared.values(), question)
+        plugin_ep = choose_plugin(prepared.values()[::-1], question)
```

Comment:
Wait, what? :O Why do you want plugins to be reversed alphabetical order?

BTW, `prepared.values()` returns stuff in random order, so this makes it reverse random order.

Also, if we really really really want to do reverse alphabetical order (which doesn't make any sense to me), unit tests are missing.